docs/spikes/using-image-data/ai-vision.ipynb

```diff
@@ -56,16 +48,27 @@
     "### Question\n",
     "\n",
     "To ask a question using this data, the following steps are performed:\n",
-    "1. Generate embeddings for the question using `text-embeddings-002-ada`\n",
-    "2. Generate image imbeddings for the question using computer vision\n",
+    "1. Generate embeddings for the question using computer vision `vectorizeText` API\n",
+    "2. Generate embeddings for the question using `text-embeddings-002-ada`\n",
     "3. Search index using both embeddings\n",
     "4. Generate blob sas url from returned search results\n",
-    "5. Pass question, along with blob sas url to `gpt-4-vision` chat completions end point"
+    "5. Pass question, along with blob sas url to `gpt-4-vision` chat completions end point\n",
+    "\n",
+    "\n",
+    "## Why do we need two different embedding models?\n",
+    "\n",
+    "It is not required to use two different embedding models, however, using both Azure Computer Vision to embed the image\n",
+    "and `gpt-4-vision` to generate a description that is then embedded by `text-embeddings-002-ada` provides richer data and\n",
+    "provides better search results. This is useful in particular for diagrams and flow charts which show relationships and\n",
+    "decision points.\n",
+    "\n",
+    "When embedding the question, we need to embed it with both embeddings models to allow us to search over the previously\n",
+    "generated vectors. This is due to the embeddings models generating differently sized and formatted vectors."
```

Comment:
Ok, so we're embedding both the image and the question with both models now.

In the implementation in CWYD, do we plan to embed all incoming questions with both models? Because when embedding the question we will not necessarily know if the answer is in an image or in a word doc for example.

Reply:
Yes that makes sense. Thanks Adam. I think that answers all my questions.